prqlc/bindings/clib/Cargo.toml

```diff
@@ -8,8 +8,13 @@ repository.workspace = true
 rust-version.workspace = true
 version.workspace = true
 
+# This means we can build with `--features=default`, which can make builds more generic
+[features]
+default = []
+
 [lib]
 crate_type = ["staticlib", "cdylib"]
+# crate_type = ["cdylib"]
```

Comment:
@eitsupi do you know what we should have here? We had had it just be `cdylib`, but then nothing is built under `musl`. Are we OK to have both?

Reply:
I don't know but maybe rust-lang/cargo#8607, rust-lang/rust#59302?
Maybe it's better to just build gnu for now?

Reply:
OK, I'll leave them both in for the moment, no harm in producing too much, and we can refine later.

We only have targets for `musl` for `prqlc` so I won't make new `gnu` ones unless we know there'll be a problem. But can move to `gnu` if anyone raises an issue...